datafusion/core/tests/tracing/mod.rs

```diff
@@ -76,7 +76,12 @@ async fn run_query() {
     let ctx = SessionContext::new();
 
     // Get the test data directory
-    let test_data = parquet_test_data();
+    let mut test_data = parquet_test_data();
+    if cfg!(target_os = "windows") {
+        // Replace backslashes (Windows paths) as they are invalid: https://datatracker.ietf.org/doc/html/rfc8089#appendix-E.4
+        // Also add "/", as Windows paths start with <Drive>:/ instead of "/"
+        test_data = format!("/{}", test_data.replace("\\", "/"))
+    }
```

Comment:
Would it be better to fix this at the root cause, aka inside `parquet_test_data()` instead of patching it after the fact here?

Reply:
I've replaced the backslashes directly in `parquet_test_data`, which should help prevent errors in new tests.
However, I still needed to prepend the path with `/` in that function, since it is passed directly to `test://`, and the URI should be `test:///C:/...` instead of `test://C:/` (this cannot be done in `parquet_test_data`, as it will make the path invalid for other tests).